qemu/tests/cfg/virtio_port_hotplug.cfg

```diff
@@ -23,13 +38,25 @@
             extra_serials = "port1 port2"
             serial_type_port1 = "virtserialport"
             serial_type_port2 = "virtserialport"
-            guest_scripts = VirtIoChannel_guest_send_receive.py;windows_support.py
-            host_script = serial_host_send_receive.py
-            guest_script = VirtIoChannel_guest_send_receive.py
-            clean_cmd = rm -f
-            guest_script_folder = /var/tmp/
-            Windows:
-                guest_script_folder = C:\
-                clean_cmd = del /f /q
-                tmp_dir = %TEMP%
-                python_bin = python2.7
+            shutdown_method = system_powerdown
+            repeat_times = 1
+            variants:
+                - @default:
+                    plug_same_nr = yes
+                - with_reboot:
+                    interrupt_test_after_unplug = reboot_guest
+                    interrupt_test_after_plug = reboot_guest
+                    reboot_method = shell
+                - with_system_reset:
+                    interrupt_test_after_unplug = reboot_guest
+                    interrupt_test_after_plug = reboot_guest
+                    reboot_method = system_reset
+                    sleep_before_reset = 0
+                - with_shutdown_after_unplug:
+                    interrupt_test_after_unplug = shutdown_guest
+                - with_shutdown_after_plug:
+                    interrupt_test_after_plug = shutdown_guest
+                - with_live_migration_after_unplug:
```

Comment:
is there no with_live_migration_after_plug?

Reply:
no this case as framework does not support.